.github/workflows/check-licenses-active-issues.yml

```diff
@@ -0,0 +1,127 @@
+# Checks for active licensing issues across all tracking branches for camunda/camunda@* FOSSA projects
+# owner: @camunda/monorepo-devops-team
+---
+name: Check Licenses (Active Issues)
+
+on:
+  schedule:
+    # Run weekly
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
```

Comment:
Would it be possible to introduce a dry-run mechanism that allows testing this workflow on PRs that change this workflow, like here:

* https://github.com/camunda/camunda/blob/main/.github/workflows/statistics-weekly.yml#L13
* https://github.com/camunda/camunda/blob/main/.github/workflows/statistics-weekly.yml#L107-L111